integration-tests/features/steps/stack_analysis.py

```diff
@@ -650,6 +650,19 @@ def check_analyzed_dependencies_count(context, expected=1):
             analyzed_dependencies_count, expected)
 
 
+@then("I should find exactly one unknown dependency")
+@then("I should find exactly {expected:n} unknown dependencies")
+def check_unknown_dependencies_count_exact_check(context, expected=1):
+    """Check number of unknown dependencies."""
+    jsondata = context.response.json()
+    assert jsondata is not None
+    path = "result/0/user_stack_info/unkwnown_dependencies_count"
+    unknown_dependencies_count = get_value_using_path(jsondata, path)
+    assert unknown_dependencies_count == expected, \
```

Comment:
Rather the test should be `unknown_dependencies_count <= expected`

Reply:
it is defined below:

`@then("I should find no more than {expected:n} unknown dependencies")`

Reply:
Ok